buildSrc/mac.gradle

```diff
@@ -214,7 +222,11 @@ MAC.font.lib = "javafx_font"
 MAC.media = [:]
 MAC.media.compiler = "${compiler} ${ccBaseFlags.join(" ")}"
 //MAC.media.ccFlags = ccBaseFlags
-MAC.media.linker = "${linker} ${commonParams.join(" ")}"
+if (IS_STATIC_BUILD) {
+    MAC.media.linker = "${linker}"
+} else {
+    MAC.media.linker = "${linker} ${commonParams.join(" ")}"
+}
```

Comment:
Is this used by media only? If yes, then I think it would better to revert it as well.